### PR TITLE
Update ingredients.txt

### PR DESCRIPTION
**Pull request by teolemon**
## Description

Describe the changes made and why they were made instead of how they were made.

## Related issues and discussion
#{Issue Number}
 

## Checklist
 
Make sure you've done all the following (You can delete the checklist before submitting)
 
 - [ ] Code is well documented
 - [ ] Include unit tests for new functionality
 - [ ] Code passes Travis builds in your branch
 - [ ] If you have multiple commits please combine them into one commit by squashing them.
 - [ ] Read and understood the [contribution guidelines](https://github.com/openfoodfacts/openfoodfacts-server/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: taxonomies/ingredients.txt [hunk 1]
```diff
@@ -277,6 +279,26 @@ fr:farine de riz blanc
 
 fr:farine de riz complet
 
+fr:vinaigre balsamique de modène
+
+fr:vinaigres d'alcool et de vin blanc
+
+fr:vinaigre de malt
+
+fr:vinaigre dialcool
```
Comment (stephanegigandet) [inline taxonomies/ingredients.txt]: dialcool -> d'alcool

File: taxonomies/ingredients.txt [hunk 2]
```diff
@@ -2025,6 +2043,9 @@ fr:millet
 fr:sirop de blé
 allergens:fr:sirop de _blé_
 
+sirop de maïs à haute teneur en fructose (SGHF, ou HFCS selon le signe anglais de high-fructose corn syrup
```
Comment (stephanegigandet) [inline taxonomies/ingredients.txt]: manque un truc ?

File: taxonomies/ingredients.txt [hunk 3]
```diff
@@ -6735,6 +6753,27 @@ fr:fond de sauce
 
 fr:érythrosine
 
+en:vegetal oil
+fr:huile végétale
```
Comment (stephanegigandet) [inline taxonomies/ingredients.txt]: i think there's another en:vegetal oil, vegetal oils
fr:huile végétale, huiles végétales that i added

File: taxonomies/ingredients.txt (hunk 2 above)
Comment (teolemon) [reply taxonomies/ingredients.txt]: j'ai été distrait :)

File: taxonomies/ingredients.txt (hunk 3 above)
Comment (teolemon) [reply taxonomies/ingredients.txt]: not sure

**Comment by teolemon** [comment]
made the changes about vegoil

**Pull request merged**